apps/vmq_server/src/vmq_mqtt5_fsm.erl

```diff
@@ -295,12 +309,8 @@ connected(#mqtt5_publish{message_id=MessageId, topic=Topic,
             dispatch_publish(QoS, MessageId, Msg, State)
     end,
     case Ret of
-        {error, not_allowed} ->
-            %% TODOv5 this will tell the client that the publish
-            %% wasn't allowed. Is it a good idea to give the client
-            %% this information or is it better to just drop the
-            %% connection?
-            terminate(not_authorized, State);
+        {error, recv_max_exceeded} ->
+            terminate(?RECEIVE_MAX_EXCEEDED, State);
```

Comment:
Can't we still get `{error, not_allowed}`  here, for instance from a plugin which doesn't allow the publish?

Reply:
I think I've covered all occurrences.